src/head.js

```diff
@@ -0,0 +1,5 @@
+function head(arr) {
+    return arr[0];
```

Comment:
Please handle the situation when somebody provides invalid argument (like null)

And write some test in `test/head.spec.js`

Reply:
Added tests and handling of invalid arguments